.travis.yml

```diff
@@ -1,25 +1,69 @@
 sudo: required
 dist: trusty
 language: python
+compiler: gcc
 
 virtualenv:
   system_site_packages: true
 
 python:
   - 3.4
 
+env:
+  global:
+     # - QT_BASE=53
+     # - QT_BASE=54
+     # - QT_BASE=55
+     # - QT_BASE=56
+     - QT_BASE=57
+     #- GCC_ARM=/opt/gcc-arm-none-eabi/bin
+  matrix:
+     # - FLAVOR=ALL
+     - FLAVOR=DEFAULT
```

Comment:
OK, well I liked saving "COMPANION" for if we ever want just a Companion build... the "DEFAULT" is actually "make all" with default settings... so it seemed to make more sense in the end.  If you're OK with it, I'll just add a comment to that effect.